src/js/profile/mobile/widget/TimePicker.js

```diff
@@ -97,6 +97,11 @@
 				self._setValue(new Date());
 
 				self.option("format", options.format);
+
+				// Calculate the spin widget after building the widget
+				Object.keys(self._spins).forEach(function (key) {
+					self._spins[key]._refresh();
```

Comment:
This works good but you use '@protected' method of widget. Please use 'refresh' instead of  `_refresh`.